### ISSUE TITLE
orcropus-gpageseg: UnboundLocalError: local variable 'gray' referenced before assignment

### ISSUE THREAD

**Issue by wrznr** [tmbdev/ocropy#222]
## Current Behavior
If I use the CLI option `--gray` and no `.nrm.png` is provided, an UnboundLocalError is thrown.

## Possible Solution
In line 380 of ocropus-gpageseg, add one indent level to the `checktype` test and an error message which complains about the missing grayscale file (since it is a prerequisite for `--gray`, cf. line 431). Also an update of the CLI option help might be useful.

I would have liked to create a pull request for that one. But it is not possible to fork the same github project twice (https://stackoverflow.com/questions/6675994/is-it-possible-to-fork-a-fork-in-github) and I already forked the fork `kraken`.

**Comment by zuphilip** [comment]
Can you create a PR from a different branch, i.e. something like
```
git remote add ocropy https://github.com/tmbdev/ocropy.git
git fetch ocropy
git checkout -b ocropy ocropy/master
```
?

**Comment by zuphilip** [comment]
Fixed in #287
